modules/associate/associate.js

```diff
@@ -43,30 +44,32 @@ exports.printAssociations = function(api, event) {
 	api.sendMessage(message, event.thread_id);
 };
 
-exports.clear = function(api, event) {
-	this.config[event.thread_id] = {};
+clear = function(api, event) {
+	exports.config[event.thread_id] = {};
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'thread_id' is not in camel case.](https://www.codacy.com/app/theprojectshop/Kassy/file/2529926667/issues/source?bid=3282987&fileBranchId=3282998#l48)